scripts/scan.sh

```
@@ -1 +1,10 @@
-# TODO: write the script
+printf "\n====== SCANNING FOR CREDIT CARD NUMBERS ======\n"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)